public/js/main.js

```diff
@@ -216,24 +217,45 @@ $(document).ready(() => {
     const B = $('#tradeCurrency').val();
     console.log(`Loading chart for ${A}_${B}...`);
     $.getJSON(`/data?currencyA=${A}&currencyB=${B}`, (data) => {
+      console.log(data);
       const ops = {};
       ops.A = A; ops.B = B; ops.data = data; chartData = data;
-      $('#hcharts-strategy').highcharts('StockChart', createHighChartsObj(ops));
+      mainChart = Highcharts.stockChart('hcharts-strategy', createHighChartsObj(ops));
+      //$('#hcharts-strategy').highcharts('StockChart', createHighChartsObj(ops));
     });
   });
   $('#addIndicatorBtn_SMA').click(() => {
     console.log('### Inside addIndicator_SMA');
     const SMAParam = $('#addIndicator_SMA').val();
+
+    // Parsing pattern for csv SMA values
+    var pattern = /\s*,\s*/;
```

Comment:
Try to avoid `var` when possible - best to use either `let` or `const` going forward